project_key/tests/test_project.py

```
@@ -69,3 +69,8 @@ def test_09_name_onchange_with_key(self):
     def test_10_generate_unique_key_with_counter(self):
         project = self.Project.create({"name": "OCA"})
         self.assertEqual(project.key, "OCA1")
+
+    def test_11_generate_unique_key_with_counter_inactive(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_11_generate_unique_key_with_counter_inactive`

Add return type annotation: `None`

(ANN201)